src/components/book.vue

```diff
@@ -1,118 +1,122 @@
+                                                </v-list-tile>
+                                            </div>
+                                        </span>        <span>
+                                            <v-subheader>Similar Books</v-subheader>
+                                            <div>
+                                                <li v-for="book in similar">
+                                                    <router-link :to="'/search/'+book.Name">{{book["Name"]}}</router-link>
+                                                </li>
 
-        </span>
-          </div>
-</v-card-title>
-<!-- @TODO FEATURES TO BE ADDED-->
-        <v-card-actions>
-          <v-btn flat color="orange">read</v-btn>
-          <v-btn flat color="orange">share</v-btn>
-</v-card-actions>
-                </div>
-                                    <div v-else-if="result.error">  
-                      <v-list-tile>
-                      <v-list-tile-content>
-                        <v-list-tile-title >No search results found in {{index}}.</v-list-tile-title>
-                      </v-list-tile-content>
-                    </v-list-tile>
-                      </div>
-                    <v-divider ></v-divider>
+                                                {{similarBooks}}
+                                            </div>
 
-              </v-card>
-            </v-flex>
-          </v-layout>
-        </v-container>
+                                        </span>
+                                    </div>
+                                </v-card-title>
+                                <!-- @TODO FEATURES TO BE ADDED-->
+                                <v-card-actions>
+                                    <v-btn flat color="orange">read</v-btn>
+                                    <v-btn flat color="orange">share</v-btn>
+                                </v-card-actions>
+                            </div>
+                            <div v-else-if="result.error">
+                                <v-list-tile>
+                                    <v-list-tile-content>
+                                        <v-list-tile-title >No search results found in {{index}}.</v-list-tile-title>
+                                    </v-list-tile-content>
+                                </v-list-tile>
+                            </div>
+                            <v-divider ></v-divider>
 
-      </v-content>
+                        </v-card>
+                    </v-flex>
+                </v-layout>
+            </v-container>
+
+        </v-content>
     </v-app>
 
 </div>
 
 
 </template>
- 
+
 <script>
 export default {
     name : 'book',
     data(){
         return {
             result: {},
             id: this.$route.params.id,
-            src: this.$route.params.src
+            src: this.$route.params.src,
+            similar: {}
         }
     },
+    computed: {
+        similarBooks() {
```

Comment:
this function should be in store/actions and called using dispatch as we are using Vuex ...